assets/js/pages/UpgradablePackagesPage/index.js

```diff
@@ -1,3 +1,3 @@
 import UpgradablePackagesPage from './UpgradablePackagesPage';
 
-export default UpgradablePackagesPage;
+export default UpgradablePackagesPage
```

Comment:
I think this should fail the `lint` step?

Reply:
grrr reactions for out prettier check, why did it green light this? :0

Reply:
Hmm eslint doesn't show this as error. Maybe, because it's formatting?